Assets/MRTK/Providers/OpenXR/Scripts/MicrosoftArticulatedHand.cs

```diff
@@ -24,17 +38,21 @@ namespace Microsoft.MixedReality.Toolkit.XRSDK.OpenXR
     /// </summary>
     [MixedRealityController(
         SupportedControllerType.ArticulatedHand,
-        new[] { Utilities.Handedness.Left, Utilities.Handedness.Right })]
+        new[] { Handedness.Left, Handedness.Right })]
     public class MicrosoftArticulatedHand : GenericXRSDKController, IMixedRealityHand
     {
         /// <summary>
         /// Constructor.
         /// </summary>
-        public MicrosoftArticulatedHand(TrackingState trackingState, Utilities.Handedness controllerHandedness, IMixedRealityInputSource inputSource = null, MixedRealityInteractionMapping[] interactions = null)
+        public MicrosoftArticulatedHand(TrackingState trackingState, Handedness controllerHandedness, IMixedRealityInputSource inputSource = null, MixedRealityInteractionMapping[] interactions = null)
             : base(trackingState, controllerHandedness, inputSource, interactions, new ArticulatedHandDefinition(inputSource, controllerHandedness))
         {
 #if MSFT_OPENXR
-            handTracker = new Preview.HandTracker(controllerHandedness == Utilities.Handedness.Left ? Preview.Handedness.Left : Preview.Handedness.Right, Preview.HandPoseType.Tracked);
+#if MSFT_OPENXR_0_2_0_OR_NEWER
+            handTracker = controllerHandedness == Handedness.Left ? HandTracker.Left : HandTracker.Right;
```

Comment:
Just confirming you are intentionally not creating a new hand tracker here unlike you did below.

Reply:
Yep! in 0.2.0, the plugin moved to a new pattern, where there are two static instances (one for the left hand, one for the right) and they're accessed globally instead of being instantiated explicitly.